rfc/2022/2022-08-10-The Console UI Will Use Material UI and Adopt Styling from the VA Design System While Isolating Plugin Styling.md

```diff
@@ -0,0 +1,149 @@
+
+The [VA Design System](https://design.va.gov/about/) provides guidelines and codes to ensure consistency in digital services across the [VA.gov](https://www.va.gov/) platform. However, its primary audiences include those building veteran-facing services and applications on [VA.gov](https://www.va.gov/), whereas the Platform Console’s primary intended audience are VFS and Platform backend engineers. More importantly, the VA Design System would need to be implemented as custom components, [which would require substantial effort to rewrite and maintain code and risks incurring technical debt](https://vfs.atlassian.net/wiki/spaces/CUI/pages/2256306184/Draft+Explore+replacing+components+in+Console+UI+with+Custom+Components). 
+
+## Motivation
+
+We need to accomplish the following goals:
+
+    1. Identify and adopt a design system strategy that will adhere to core design principles of the VA Design System while making it easy for the Platform Console to apply updates from the open-source project upon which the software is based.
+
+    2. Document a strategy that allows plugin developers to adopt, extend, or opt-out of the design system Console UI is using within the confines of their plugin.
+
+    3. Define a process for improving the accessibility of the product's chosen design system to meet or exceed WCAG 2.0 AA standards.
+
+## Design
+
+We will adopt the Material UI + VA Design System Styling utilized by the Lighthouse team, what they have dubbed the **VA Enterprise Design System**. This system will comply with all of the brand identity and accessibility standards of the VA Design System. The significant difference is that the VA Enterprise Design system has a different end-user base internal to the organization and focuses on software engineering. 
+
+We will document how to adopt this system for plugin developers that need to extend the Console UI and don't need additional UI/UX flexibility. In addition, we will document how to extend the system for plugin developers who need to offer additional capabilities for visual content presentation within Console UI. We will also document how to disable all of the VA Enterprise Design System for plugin developers that need highly customized UI/UX or the need to demonstrate another design system, such as The VA Design System.
+
+This will 1) allow the Console documentation and use to begin faster than anticipated, and 2) contribute to the formation of an internal “enterprise” design system so that non-veteran-facing tools can also remain consistent.
```

Comment:
We on the Design System Team would love to talk with you all about this and figure out how we could support your team better.

Reply:
Rolled comment into document as it wasn't directly addressed.